Mathlib/Topology/Algebra/UniformGroup/Defs.lean

```diff
@@ -442,8 +442,7 @@ theorem tendsto_div_comap_self (de : IsDenseInducing e) (x₀ : α) :
   have comm : ((fun x : α × α => x.2 / x.1) ∘ fun t : β × β => (e t.1, e t.2)) =
       e ∘ fun t : β × β => t.2 / t.1 := by
     ext t
-    change e t.2 / e t.1 = e (t.2 / t.1)
-    rw [← map_div e t.2 t.1]
```

Comment:
This one can probably just become `exact (map_div e t.2 t.1).symm`

Reply:
But `simp` is better, no?